pkg/server/ca/manager_test.go

```diff
@@ -331,6 +346,10 @@ func (m *ManagerTestSuite) requireKeypairSetKeysEqual(set1, set2 *keypairSet) {
 }
 
 func (m *ManagerTestSuite) requireKeypairSetKeysNotEqual(set1, set2 *keypairSet) {
+	m.Require().NotNil(set1)
+	m.Require().NotNil(set1.x509CA)
+	m.Require().NotNil(set2)
+	m.Require().NotNil(set2.x509CA)
```

Comment:
do we want to check `jwtSigningKey` too? We access it below

Reply:
sure. i can do that.